var/spack/repos/builtin/packages/py-shapely/package.py

```diff
@@ -11,16 +11,25 @@ class PyShapely(PythonPackage):
     """
 
     homepage = "https://github.com/Toblerity/Shapely"
-    url      = "https://pypi.io/packages/source/S/Shapely/Shapely-1.6.4.tar.gz"
+    url      = "https://pypi.io/packages/source/S/Shapely/Shapely-1.6.4.post2.tar.gz"
 
+    maintainers = ['adamjstewart']
+    import_modules = [
+        'shapely', 'shapely.geometry', 'shapely.algorithms',
+        'shapely.examples', 'shapely.speedups', 'shapely.vectorized',
+    ]
+
+    version('1.6.4.post2', sha256='c4b87bb61fc3de59fc1f85e71a79b0c709dc68364d9584473697aad4aa13240f')
     version('1.6.4', '7581ef2d0fb346f9ed157f3efc75f6a4')
 
-    depends_on('python@2.6.0:2.8,3.4.0:', type=('build', 'run'))
-    depends_on('py-setuptools',     type='build')
-    depends_on('py-cython@0.19:',   type='build')
-    depends_on('py-numpy@1.4.1:',   type=('build', 'run'))
-    depends_on('geos@3.3:')
+    depends_on('python@2.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-cython', type='build')
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('geos')
+    depends_on('geos@3.3:', when='@1.3:')
+    depends_on('py-pytest', type='test')
 
     def setup_environment(self, spack_env, run_env):
-        spack_env.set('GEOS_CONFIG', self.spec['geos'].prefix)
-        spack_env.prepend_path('LD_LIBRARY_PATH', self.spec['geos'].prefix.lib)
```

Comment:
How do you know removing this line won't break something?

Reply:
Because I built it successfully. And because Spack adds that dir via the compiler wrappers.